Wiki restructuring - interoperability (#1074)
Azure/ResourceModules: docs/wiki/Interoperability - Transition to Azure DevOps Pipelines.md

> Not all customers may be using GitHub repos and/or GitHub Actions. CARML is able to be hosted in a GitHub repo and deployed with Azure DevOps Pipelines or it can completely be ran out of Azure DevOps.

Reviewer: ```suggestion
Not all customers may be using GitHub repos and/or GitHub Actions. CARML can be hosted in a GitHub repo and deployed using Azure DevOps Pipelines or run entirely through Azure DevOps.
```
